script/test

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# script/test: Run test suite for application. Optionally pass in a path to an
+#              individual test file to run a single test.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+echo "Linting files..."
```

Comment:
I think we typically use the format `==> message` when `echo`ing in `script/` files. [script/bootstrap](https://github.com/dxw/playbook/blob/main/script/bootstrap) and [script/setup](https://github.com/dxw/playbook/blob/main/script/setup) have a few examples